Gifski/Tooltip.swift

```diff
@@ -0,0 +1,93 @@
+import AppKit
+
+final class Tooltip: NSPopover {
+	init(text: String, closeOnClick: Bool = true, contentInsets: NSEdgeInsets = .zero, maxWidth: CGFloat? = nil) {
+		super.init()
+		setupContent(text: text, closeOnClick: closeOnClick, contentInsets: contentInsets, maxWidth: maxWidth)
+	}
+
+	required init?(coder: NSCoder) {
+		super.init(coder: coder)
+		setupContent(text: "", closeOnClick: true, contentInsets: .zero, maxWidth: nil)
+	}
+
+	private func setupContent(text: String, closeOnClick: Bool, contentInsets: NSEdgeInsets, maxWidth: CGFloat?) {
+		contentViewController = ToolTipViewController(text: text, contentInsets: contentInsets, maxWidth: maxWidth) { [weak self] in
+			if closeOnClick {
+				self?.close()
+			}
+		}
+		behavior = closeOnClick ? .transient : .applicationDefined
```

Comment:
`transient` => `semitransient`

Clicking outside the app should not close the tooltip.